### PR TITLE
[zh]Sync content/zh/examples/controllers/fluentd-daemonset.yaml

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh/examples/controllers/fluentd-daemonset-update.yaml [hunk 1]
```diff
@@ -19,9 +19,13 @@ spec:
         name: fluentd-elasticsearch
     spec:
       tolerations:
-      # this toleration is to have the daemonset runnable on master nodes
-      # remove it if your masters can't run pods
+       # 这些容忍是让守护进程在控制平面节点上运行
```
Comment (tengqm) [inline content/zh/examples/controllers/fluentd-daemonset-update.yaml:22]:
```suggestion
       # 这些容忍度使得 DaemonSet  能够在控制平面节点上运行
```

File: content/zh/examples/controllers/fluentd-daemonset-update.yaml [hunk 2]
```diff
@@ -19,9 +19,13 @@ spec:
...
+       # 如果你的控制节点不应该运行 Pod，可以删除它们
```
Comment (tengqm) [inline content/zh/examples/controllers/fluentd-daemonset-update.yaml:23]:
```suggestion
       # 如果你的控制节点不可以运行 Pod，可以删除它们
```

File: content/zh/examples/controllers/fluentd-daemonset.yaml [hunk 3]
```diff
@@ -19,9 +19,13 @@ spec:
         name: fluentd-elasticsearch
     spec:
       tolerations:
-      # this toleration is to have the daemonset runnable on master nodes
-      # remove it if your masters can't run pods
+       # 这些容忍是让守护进程在控制平面节点上运行
```
Comment (tengqm) [inline content/zh/examples/controllers/fluentd-daemonset.yaml:22]:
```suggestion
       # 这些容忍度使得 DaemonSet 能够在控制平面节点上运行
```

File: content/zh/examples/controllers/fluentd-daemonset.yaml [hunk 4]
```diff
@@ -19,9 +19,13 @@ spec:
...
+       # 如果你的控制节点不应该运行 Pod，可以删除它们
```
Comment (tengqm) [inline content/zh/examples/controllers/fluentd-daemonset.yaml:23]:
```suggestion
       # 如果你的控制节点不可以运行 Pod，可以删除它们
```

**Comment by Sea-n** [comment]
One of files in this PR is conflicted with #33681.
- #33681

Please rebase so that we can review and merge your change to another file.

**Pull request closed without merging**